orange-datamodel/src/main/java/com/hartwig/hmftools/datamodel/linx/LinxReportableReason.java

```diff
@@ -0,0 +1,32 @@
+package com.hartwig.hmftools.datamodel.linx;
+
+public enum LinxReportableReason
+{
+    OK(""),
+    NOT_KNOWN("Unknown fusion pair"),
+    UNPHASED_NOT_KNOWN("Unphased, no known fusion pair"),
+    UNPHASED_5P_UTR("Unphased, 5P UTR"),
+    UNPHASED_SHORT("Unphased, short unphased distance"),
+    SGL_NOT_KNOWN("SGL, no known fusion pair"),
+    PRE_GENE_DISTANCE("Max upstream distance"),
+    NMD("Nonsense mediated decay"),
+    NEG_SPLICE_ACC_DISTANCE("Negative previous splice acceptor distance"),
+    EXON_SKIPPING("Exon skipping"),
+    CHAIN_TERMINATED("Chain terminated"),
+    NON_DISRUPTIVE_CHAIN("Non-disruptive chain"),
+    INVALID_TRAVERSAL("Invalid chain traversal"),
+    CHAIN_LINKS("Maximum chain links"),
+    PROTEIN_DOMAINS("Protein domains");
+
+    private final String display;
```

Comment:
@kduyvesteyn What do you mean by a "user of the datamodel"? :)

I did not consider to implement it another way, since I saw it was implemented like this for `ChordStatus`.

Reply:
If you implement a `display` in the datamodel as you've done you sort of "enforce" common display across users (e.g. ORANGE and OncoAct and ACTIN reports would (probably) use the display function from the datamodel). 

You can also not provide a display function, in which ORANGE, OncoAct, ACTIN have to implement their own (but can decide to differ). 

There is no right or wrong here, though it is good practice to keep datamodels small and simple (e.g. all other enums in the linx package have no display). Also, it is sort of "arrogant" to assume the datamodel knows exactly how to display values in every possible use-case. 

Hope this makes sense somewhat :) And as mentioned - you can go both ways, just make sure you understand the trade-off!

Reply:
This is very useful feedback, thanks! I didn't think about it this way. I removed the display logic from the datamodel and moved it to `DnaFusionTable`, which I think is the most suitable place for the display mapping?

Reply:
Makes sense to me, the display is now right next to the actual rendering code which makes it more readable in my opinion!